django_tables2/columns/base.py

```diff
@@ -434,7 +434,7 @@ def verbose_name(self):
 
         # This is our reasonable fallback, should the next section not result
         # in anything useful.
-        name = title(self.name.replace('_', ' '))
+        name = self.name.replace('_', ' ')
```

Comment:
If this change is applied, the second part of the first sentence of the docstring of this method is not true anymore.